sdk/lib/rtl/network.c

```diff
@@ -515,7 +515,7 @@ RtlIpv6AddressToStringA(
     ASSERT(Result >= Buffer);
     ASSERT(Result < Buffer + RTL_NUMBER_OF(Buffer));
 
-    Status = RtlUnicodeToMultiByteN(S, RTLIPV6A2S_MAX_LEN, NULL, Buffer, (wcslen(Buffer) + 1) * sizeof(WCHAR));
+    Status = RtlUnicodeToMultiByteN(S, RTLIPV6A2S_MAX_LEN, NULL, Buffer, (ULONG)(wcslen(Buffer) + 1) * sizeof(WCHAR));
```

Comment:
No extensive check about whether some `Length = wcslen(Buffer);` could be larger than ULONG_MAX? (as done below)

Reply:
No, and the one below is actually useless, because the whole thing is a static buffer with known size. Also RtlIpv6AddressToStringExW already returns the length, so there is no need to call wcslen again.